Trackovid19-web/package.json

```diff
@@ -72,8 +72,7 @@
   },
   "lint-staged": {
     "*.{js,json,css,scss,less,md,ts,html,component.html}": [
-      "prettier --write",
-      "git add"
```

Comment:
Can you give more context why you removed this?

Reply:
Since version 10 of `lint-staged` library, the `git add` isn't required anymore 🙂.

Reply:
👍 Thank you